src/platform/plugins/shared/unified_doc_viewer/public/components/doc_viewer_table/table.tsx

```diff
@@ -511,6 +513,12 @@ export const DocViewerTable = ({
           css={css`
             min-block-size: 0;
             display: block;
+            .euiDataGridRow {
+              &:hover {
+                // we keep using a deprecated shade until proper token is available
```

Comment:
As per [this thread](https://elastic.slack.com/archives/C7QC1JV6F/p1738776880098029?thread_ts=1738767561.281099&cid=C7QC1JV6F)

Reply:
Thanks for checking! In Amsterdam `lightestShade` looks to be the same colour as the striped table rows, but I think that's ok for now, and it looks nice in Borealis.

Now my only remaining request would be to also update the Discover grid to the same hover styles for consistency. I think we should either use `euiTheme.colors.lightestShade` or `euiTheme.colors.backgroundBaseInteractiveHover` for both, but not a mixture. The Discover grid styles can be found in `src/platform/packages/shared/kbn-unified-data-table/src/components/data_table.scss`.

Reply:
Agree, changing!